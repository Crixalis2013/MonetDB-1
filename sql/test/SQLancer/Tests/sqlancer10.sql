START TRANSACTION;
CREATE TABLE "sys"."t0" ("c0" TIME NOT NULL, "c1" VARCHAR(143),
	CONSTRAINT "t0_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t0_c0_unique" UNIQUE ("c0"), CONSTRAINT "t0_c1_unique" UNIQUE ("c1"));
COPY 7 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
21:19:08	""
13:02:49	NULL
01:02:11	NULL
16:34:25	NULL
12:11:43	NULL
10:35:38	NULL
04:26:50	NULL

CREATE TABLE "sys"."t1" ("c0" CHAR(375) NOT NULL, CONSTRAINT "t1_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t1_c0_fkey" FOREIGN KEY ("c0") REFERENCES "sys"."t0" ("c1"));
insert into t1 values ('');
insert into t1(c0) values ((select 'a')), ('b');
insert into t1(c0) values(r']BW扗}FUp'), (cast((values (greatest(r'Aᨐ', r'_'))) as string(616))), (r'');
ROLLBACK;
