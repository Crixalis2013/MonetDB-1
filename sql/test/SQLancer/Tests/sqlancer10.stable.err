stderr of test 'sqlancer10` in directory 'sql/test/SQLancer` itself:


# 13:59:46 >  
# 13:59:46 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-215607" "--port=30526"
# 13:59:46 >  


# 13:59:46 >  
# 13:59:46 >  "Done."
# 13:59:46 >  

