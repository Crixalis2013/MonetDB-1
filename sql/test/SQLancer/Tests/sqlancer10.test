statement ok
START TRANSACTION

statement ok
CREATE TABLE "sys"."t0" ("c0" TIME NOT NULL, "c1" VARCHAR(143),
	CONSTRAINT "t0_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t0_c0_unique" UNIQUE ("c0"), CONSTRAINT "t0_c1_unique" UNIQUE ("c1"))

statement ok
COPY 7 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"'
<COPY_INTO_DATA>
21:19:08	""
13:02:49	NULL
01:02:11	NULL
16:34:25	NULL
12:11:43	NULL
10:35:38	NULL
04:26:50	NULL

statement ok
CREATE TABLE "sys"."t1" ("c0" CHAR(375) NOT NULL, CONSTRAINT "t1_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t1_c0_fkey" FOREIGN KEY ("c0") REFERENCES "sys"."t0" ("c1"))

statement ok
insert into t1 values ('')

statement ok
insert into t1(c0) values ((select 'a')), ('b')

statement ok
insert into t1(c0) values(r']BW扗}FUp'), (cast((values (greatest(r'Aᨐ', r'_'))) as string(616))), (r'')
