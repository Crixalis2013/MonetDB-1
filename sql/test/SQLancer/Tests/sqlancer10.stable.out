stdout of test 'sqlancer10` in directory 'sql/test/SQLancer` itself:


# 13:59:46 >  
# 13:59:46 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-215607" "--port=30526"
# 13:59:46 >  

#START TRANSACTION;
#CREATE TABLE "sys"."t0" ("c0" TIME NOT NULL, "c1" VARCHAR(143),
#	CONSTRAINT "t0_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t0_c0_unique" UNIQUE ("c0"), CONSTRAINT "t0_c1_unique" UNIQUE ("c1"));
#COPY 7 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#21:19:08	""
#13:02:49	NULL
#01:02:11	NULL
#16:34:25	NULL
#12:11:43	NULL
#10:35:38	NULL
#04:26:50	NULL
[ 7	]
#CREATE TABLE "sys"."t1" ("c0" CHAR(375) NOT NULL, CONSTRAINT "t1_c0_pkey" PRIMARY KEY ("c0"), CONSTRAINT "t1_c0_fkey" FOREIGN KEY ("c0") REFERENCES "sys"."t0" ("c1"));
#insert into t1 values ('');
[ 1	]
#ROLLBACK;

# 13:59:46 >  
# 13:59:46 >  "Done."
# 13:59:46 >  

